---
layout: page
title: Publications
---

Check out [google scholar](http://scholar.google.com/citations?user=uxSj18QAAAAJ&hl=en) for citation statistics (and more).

For small corrections to published papers, see [this page](2021/10/06/Corrigenda.html).

### UCSD

#### Preprints and  accepted conference papers
0. Stepanauskas, Ramunas, Julia M Brown, Uyen Mai, Oliver Bezuidt, Maria Pachiadaki, Joe Brown, Steven Biller, Paul M Berube, Nicholas R Record, and Siavash Mirarab. “Gene Exchange Networks Define Species-like Units in Marine Prokaryotes.” BioRxiv, 2020, 2020.09.10.291518. [doi:10.1101/2020.09.10.291518](https://doi.org/10.1101/2020.09.10.291518).
0. Tabaghi, Puoya,  Michael Khanzadeh, Yusu Wang, Siavash Mirarab. “Principal Component Analysis in Space Forms.” arXiv. 2023.  [doi:10.48550/arxiv.2301.02750](https://doi.org/10.48550/arxiv.2301.02750). 
0. Jiang, Yueyu, Daniel McDonald, Rob Knight, and Siavash Mirarab. “Scaling Deep Phylogenetic Embedding to Ultra-Large Reference Trees: A Tree-Aware Ensemble Approach.” BioRxiv, January 1, 2023, 2023.03.27.534201. [doi:10.1101/2023.03.27.534201](https://doi.org/10.1101/2023.03.27.534201).
0. Shayesteh Arasti, Puoya Tabaghi, Yasamin Tabatabaee, and Siavash Mirarab. “Optimal Tree Metric Matching Enables Phylogenomic Branch Length Reconciliation.” bioRxiv, January 1, 2023, 2023.11.13.566962. <https://doi.org/10.1101/2023.11.13.566962>.
	0. To appear in RECOMB 2024
0. Ali Osman Berk Sapci and Siavash Mirarab. “Memory-Bound k-Mer Selection for Large Evolutionary Diverse Reference Libraries.” bioRxiv, January 1, 2024, 2024.02.12.580015. <https://doi.org/10.1101/2024.02.12.580015>.
	0. To appear in RECOMB 2024
0. Zhang, Chao, Rasmus Nielsen, and Siavash Mirarab. “CASTER: Direct Species Tree Inference from Whole-Genome Alignments.” bioRxiv, 2023. <https://doi.org/10.1101/2023.10.04.560884>.



#### Book Chapters
0. Mirarab, Siavash. “Species Tree Estimation Using ASTRAL: Practical Considerations.” In Species Tree Inference: A Guide to Methods and Applications. Princeton University Press, 2023.
	* Preprint available on [arXiv:1904.03826](https://arxiv.org/pdf/1904.03826.pdf)
0. Warnow, Tandy, and Siavash Mirarab. “Multiple Sequence Alignment for Large Heterogeneous Datasets Using SATé, PASTA, and UPP.” In Multiple Sequence Alignment. Methods in Molecular Biology, Vol 2231, 99–119, 2021. [doi:10.1007/978-1-0716-1036-7_7](https://doi.org/10.1007/978-1-0716-1036-7_7).
0. Zhu, Qiyun, and Siavash Mirarab. “Assembling a Reference Phylogenomic Tree of Bacteria and Archaea by Summarizing Many Gene Phylogenies.” In Environmental Microbial Evolution: Methods and Protocols, 137–65. New York, NY: Springer US, 2022. [doi:0~10.1007/978-1-0716-2691-7_7.11~](https://doi.org/10.1007/978-1-0716-2691-7_7.1)


#### Published Papers (including in press)

{:start="99"}
0. Şapcı, Ali Osman Berk, Eleonora Rachtman, and Siavash Mirarab. “CONSULT-II: Accurate Taxonomic Identification and Profiling Using Locality-Sensitive Hashing.” Bioinformatics, 2024, btae150. <https://doi.org/10.1093/bioinformatics/btae150>.
	* Şapcı, Ali Osman Berk, Eleonora Rachtman, and Siavash Mirarab. “CONSULT-II: Taxonomic Identification Using Locality Sensitive Hashing.” In Comparative Genomics, RECOMB-CG, Lecture Notes in Computer Science(13883). 196–214. Cham: Springer Nature Switzerland, 2023. [doi:10.1007/978-3-031-36911-7_13](https://doi.org/10.1007/978-3-031-36911-7_13)
		* Best paper award. 
0. Arasti, Shayesteh, and Siavash Mirarab. “Median Quartet Tree Search Algorithms Using Optimal Subtree Prune and Regraft.” Algorithms for Molecular Biology 19, no. 1 (March 13, 2024): 12. [doi:10.1186/s13015-024-00257-3](https://rdcu.be/dA8WU).
	* Arasti, Shayesteh, and Siavash Mirarab, “Optimal Subtree Prune and Regraft for Quartet Score in Sub-Quadratic Time,” in 23rd International Workshop on Algorithms in Bioinformatics (WABI 2023), vol. 273, Leibniz International Proceedings in Informatics (LIPIcs) (Dagstuhl, Germany: Schloss Dagstuhl – Leibniz-Zentrum für Informatik, 2023), 4:1-4:20, [doi:10.4230/LIPIcs.WABI.2023.4](https://doi.org/10.4230/LIPIcs.WABI.2023.4).
0. Balaban, Metin, Yueyu Jiang, Qiyun Zhu, Daniel McDonald, Rob Knight, and Siavash Mirarab. “Generation of Accurate, Expandable Phylogenomic Trees with UDance.” Nature Biotechnology (2023). [doi:10.1038/s41587-023-01868-8](https://doi.org/10.1038/s41587-023-01868-8). [Direct link](https://rdcu.be/dhUm5). 
0. McDonald, Daniel, Yueyu Jiang, Metin Balaban, Kalen Cantrell, Qiyun Zhu, Antonio Gonzalez, James T. Morton, Giorgia Nicolaou, Donovan H. Parks, Søren Karst, Mads Albertsen, Phil Hugenholtz, Todd DeSantis, Siavash Mirarab, Rob Knight. “Greengenes2 Unifies Microbial Data in a Single Reference Tree.” Nature Biotechnology (2023). [doi:s41587-023-01845-1](https://doi.org/10.1038/s41587-023-01845-1). [direct link](https://urldefense.com/v3/__https://rdcu.be/dhUm5__;!!Mih3wA!HRDqcF5Lf-ZEWhr9kfUkPYrHwFZye4VfwO98gZFXWo-APxho8A4TmPID7hBNYiTrK7Ww_mXLaqMwRIDeGATzjWkKYETo9NmIMA$)
0. Tabatabaee, Yasamin, Chao Zhang, Tandy Warnow, and Siavash Mirarab. “Phylogenomic Branch Length Estimation Using Quartets.” Bioinformatics 39, no. Supplement_1 (2023): i185–93. [doi:10.1093/bioinformatics/btad221](https://doi.org/10.1093/bioinformatics/btad221).
	* As part of ISMB 2023. 
0. Mai, Uyen, Eduardo Charvel, and Siavash Mirarab. “Expectation-Maximization Enables Phylogenetic Dating under a Categorical Rate Model,” bioRxiv. [doi:10.1101/2022.10.06.511147](https://doi.org/10.1101/2022.10.06.511147).
	* Presented in RECOMB 2023 and appeared as [short abstract](https://link.springer.com/content/pdf/bbm:978-3-031-29119-7/1?pdf=chapter%20toc).
0. Elghraoui, Afif, Siavash Mirarab, Krister M Swenson, and Faramarz Valafar. “Evaluating Impacts of Syntenic Block Detection Strategies on Rearrangement Phylogeny Using Mycobacterium Tuberculosi Isolates.”  Bioinformatics 39, no. 1 (2023): btad024. [doi:10.1093/bioinformatics/btad024](https://doi.org/10.1093/bioinformatics/btad024).
	* Also presented as part of RECOMB-CG 2021
0. Zhang, Chao, Andrey V. Bzikadze, Yana Safonova, and Siavash Mirarab. “A Scalable Model for Simulating Multi-Round Antibody Evolution and Benchmarking of Clonal Tree Reconstruction Methods.” Frontiers in Immunology 13 (2022): 1014439. [doi:10.3389/fimmu.2022.1014439](https://www.frontiersin.org/articles/10.3389/fimmu.2022.1014439).
0. Zhang, Chao, and Siavash Mirarab. “Weighting by Gene Tree Uncertainty Improves Accuracy of Quartet-Based Species Trees.” Edited by Aya Takahashi. Molecular Biology and Evolution 39, no. 12 (2022): msac215. [doi:10.1093/molbev/msac215](https://academic.oup.com/mbe/advance-article-abstract/doi/10.1093/molbev/msac215/6750035?utm_source=advanceaccess&utm_campaign=mbe).
0. Zhang, Chao, and Siavash Mirarab. “ASTRAL-Pro 2: Ultrafast Species Tree Reconstruction from Multi-Copy Gene Family Trees.” Bioinformatics, btac620, (2022), [doi:10.1093/bioinformatics/btac620](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btac620/6696205).
0. Balaban, Metin, Nishat A Bristy, Ahnaf Faisal, Md. Shamsuzzoha Bayzid, and Siavash Mirarab. “Genome-Wide Alignment-Free Phylogenetic Distance Estimation under a No Strand-Bias Model.” Edited by Thomas Lengauer. Bioinformatics Advances 2, no. 1 (2022): vbac055. <https://doi.org/10.1093/bioadv/vbac055>.
0. Maryam Rabiee, and Siavash Mirarab. “QuCo: Quartet-based Co-estimation of Species Trees and Gene Trees”. Bioinformatics38, no. Supplement_1 (2022): i413–21. [10.1093/bioinformatics/btac265](https://academic.oup.com/bioinformatics/article/38/Supplement_1/i413/6617531)
	* As part of ISMB 2022. 
0. Jiang, Yueyu, Puoya Tabaghi, and Siavash Mirarab. “Learning Hyperbolic Embedding for Phylogenetic Tree Placement and Updates.” Biology 11, no. 9 (2022): 1256. [doi:10.3390/biology11091256](https://doi.org/10.3390/biology11091256).
	* Jiang, Yueyu, Puoya Tabaghi, and Siavash Mirarab. “Phylogenetic Placement Problem: A Hyperbolic Embedding Approach.” In Comparative Genomics, edited by Lingling Jin and Dannie Durand, 68–85. Cham: Springer International Publishing, 2022. [doi:10.1007/978-3-031-06220-9_5](https://doi.org/10.1007/978-3-031-06220-9_5).
0. Hasan, Navid Bin, Metin Balaban, Avijit Biswas, Md. Shamsuzzoha Bayzid, and Siavash Mirarab. “Distance-Based Phylogenetic Placement with Statistical Support.” Biology 11, no. 8 (2022): 1212. [doi:10.3390/biology11081212](https://doi.org/10.3390/biology11081212).
	* Hasan, Navid Bin, Avijit Biswas, Metin Balaban, Siavash Mirarab, and Md. Shamsuzzoha Bayzid. “Fast and Accurate Branch Support Calculation for Distance-Based Phylogenetic Placements.” In Comparative Genomics, edited by Lingling Jin and Dannie Durand, 33–51. Cham: Springer International Publishing, 2022. [doi:10.1007/978-3-031-06220-9_3](https://doi.org/10.1007/978-3-031-06220-9_3).
0. Swenson, Krister M, Afif Elghraoui, Faramarz Valafar, Siavash Mirarab, and Mathias Weller. “Quantifying Hierarchical Conflicts in Homology Statements.” In Comparative Genomics, edited by Lingling Jin and Dannie Durand, 146–67. Cham: Springer International Publishing, 2022. [doi:10.1007/978-3-031-06220-9_9](https://doi.org/10.1007/978-3-031-06220-9_9).
0. Rachtman, Eleonora, Shahab Sarmashghi, Vineet Bafna, and Siavash Mirarab. “Uncertainty Quantification Using Subsampling for Assembly-Free Estimates of Genomic Distance and Phylogenetic Relationships.” Cell Systems 13, no. 10 (2022): 817-829.e3. [doi:10.1016/j.cels.2022.06.007](https://authors.elsevier.com/a/1fxla8YyDfj7HI).
	*  Abstract appeared in [RECOMB 2022](https://link.springer.com/chapter/10.1007/978-3-031-04749-7_30)
0. Jiang, Yueyu, Metin Balaban, Qiyun Zhu, and Siavash Mirarab. “DEPP: Deep Learning Enables Extending Species Trees Using Single Genes.” Systematic biology. syac031. 2022. [doi:10.1093/sysbio/syac031/6575921](https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syac031/6575921?guestAccessKey=00d69a2a-f346-4bab-9bd3-b64ddbf7627d).
	*  Abstract appeared in [RECOMB 2021](https://www.recomb2021.org/)
0. Mai, Uyen, and Siavash Mirarab. “Completing Gene Trees without Species Trees in Sub-Quadratic Time.” Bioinformatics 38, no. 6 (2022): 1532–41. [doi:10.1093/bioinformatics/btab875](https://doi.org/10.1093/bioinformatics/btab875).
0. Balaban, Metin, Yueyu Jiang, Daniel Roush, Qiyun Zhu, and Siavash Mirarab. “Fast and Accurate Distance‐based Phylogenetic Placement Using Divide and Conquer.” Molecular Ecology Resources 22, no. 3 (2022): 1213–27. [doi:10.1111/1755-0998.13527](https://doi.org/10.1111/1755-0998.13527).
0. Sarmashghi, Shahab, Metin Balaban, Eleonora Rachtman, Behrouz Touri, Siavash Mirarab, and Vineet Bafna. “Estimating Repeat Spectra and Genome Length from Low-Coverage Genome Skims with RESPECT.” PLOS Computational Biology 17, no. 11 (2021): e1009449. [doi:10.1371/journal.pcbi.1009449](https://doi.org/10.1371/journal.pcbi.1009449).
	* Abstract to appear in [RECOMB 2021](https://www.recomb2021.org/)
0. Mirarab, Siavash, Luay Nakhleh, and Tandy Warnow. 2021. “Multispecies Coalescent: Theory and Applications in Phylogenetics.” Annual Review of Ecology, Evolution, and Systematics 52, no. 1 (2021): 247–68. [doi:10.1146/annurev-ecolsys-012121-095340](https://doi.org/10.1146/annurev-ecolsys-012121-095340).
0. Rachtman, Eleonora, Vineet Bafna, and Siavash Mirarab. “CONSULT: Accurate Contamination Removal Using Locality-Sensitive Hashing.” NAR Genomics and Bioinformatics 3, no. 3 (2021): 10.1101/2021.03.18.436035. [doi:10.1093/nargab/lqab071](https://academic.oup.com/nargab/article/doi/10.1093/nargab/lqab071/6342218?guestAccessKey=713a4334-44c0-446d-a7ba-73f47ba9ecd9).
	* Also accepted and presented at [RECOMB-seq, 2021](https://www.recomb2021.org/recomb-seq-satellite).
0. Zhang, Chao, Yiming Zhao, Edward Louis Braun, and Siavash Mirarab. “TAPER: Pinpointing Errors in Multiple Sequence Alignments despite Varying Rates of Evolution.” Methods in Ecology and Evolution 12, no. 11 (2021), 2145-58. [doi:10.1111/2041-210X.13696](https://doi.org/10.1111/2041-210X.13696). [Direct Link](https://onlinelibrary.wiley.com/share/author/REMXRNH3BACNHAEWNEZS?target=10.1111/2041-210X.13696).
0. Moshiri, Niema, Davey M Smith, and Siavash Mirarab. “HIV Care Prioritization Using Phylogenetic Branch Length.” JAIDS Journal of Acquired Immune Deficiency Syndromes (2020): 2019.12.20.885202. [doi:10.1097/QAI.000000000000261](https://doi.org/10.1097/QAI.0000000000002612).
0. Feng, Shaohong, Josefin Stiller, Yuan Deng, Joel Armstrong, Qi Fang, Andrew Hart Reeve, Duo Xie, et al. “Dense Sampling of Bird Diversity Increases Power of Comparative Genomics.” Nature 587, no. 7833 (2020): 252–57. [doi:10.1038/s41586-020-2873-9](https://doi.org/10.1038/s41586-020-2873-9).
0. Rabiee, Maryam, and Siavash Mirarab. “SODA: Multi-Locus Species Delimitation Using Quartet Frequencies.” Bioinformatics, (2021), btaa1010. [doi:10.1093/bioinformatics/btaa1010](https://doi.org/10.1093/bioinformatics/btaa1010).
0. Mai, Uyen, and Siavash Mirarab. “Log Transformation Improves Dating of Phylogenies.” Molecular Biology and Evolution, msaa222 (2020). [doi:10.1093/molbev/msaa222](https://doi.org/10.1093/molbev/msaa222).
	* Abstract appeared in [RECOMB 2020](https://doi.org/10.1007/978-3-030-45257-5_31)
0. Bohmann, Kristine, Siavash Mirarab, Vineet Bafna, and M. Thomas P. Gilbert. “Beyond DNA Barcoding: The Unrealized Potential of Genome Skim Data in Sample Identification.” Molecular Ecology 29, no. 14 (2020): 2521–34. [doi:10.1111/mec.15507](https://doi.org/10.1111/mec.15507).
0. Metin Balaban, and Siavash Mirarab. “Phylogenetic Double Placement of Mixed Samples.” Bioinformatics 36, no. Supplement 1 (2020):i335–43. [doi:10.1093/bioinformatics/btaa489](https://academic.oup.com/bioinformatics/article/36/Supplement_1/i335/5870522?guestAccessKey=3e638572-1004-49c8-aeb5-b98e1a4b4735).
	* As part of ISMB 2020.
0. Ekin Tilic, Erfan Sayyari, Josefin Stiller, Siavash Mirarab, and Greg W Rouse. “More Is Needed—Thousands of Loci Are Required to Elucidate the Relationships of the ‘Flowers of the Sea’ (Sabellida, Annelida).” Molecular Phylogenetics and Evolution 151 (2020): 106892. [10.1016/j.ympev.2020.106892](https://doi.org/10.1016/j.ympev.2020.106892).
0. Zhang, Chao, Celine Scornavacca, Erin K Molloy, and Siavash Mirarab. “ASTRAL-Pro: Quartet-Based Species-Tree Inference despite Paralogy.” Molecular Biology and Evolution 37, no. 11 (2020): 3292–3307. [doi:10.1093/molbev/msaa139](https://doi.org/10.1093/molbev/msaa139).
0. Asnicar, Francesco, Andrew Maltez Thomas, Francesco Beghini, Claudia Mengoni, Serena Manara, Paolo Manghi, Qiyun Zhu, Bolzan, Mattia Fabio Cumbo, Uyen May, Jon G Sanders, Moreno Zolfo, Evguenia Kopylova, Edoardo Pasolli, Rob Knight, Siavash Mirarab, Curtis Huttenhower, Nicola Segata. “Precise Phylogenetic Analysis of Microbial Isolates and Genomes from Metagenomes Using PhyloPhlAn 3.0.” Nature Communications 11, no. 1 (2020): 2500. [doi:10.1038/s41467-020-16366-7](https://doi.org/10.1038/s41467-020-16366-7).
0. Balaban, Metin, Shahab Sarmashghi, and Siavash Mirarab. “APPLES: Scalable Distance-Based Phylogenetic Placement with or without Alignments.” Systematic Biology 69, no. 3 (2020): 566–78.[doi:10.1093/sysbio/syz063](https://academic.oup.com/sysbio/article/69/3/566/5572672?guestAccessKey=95318e8a-039e-4e92-b954-b58f64fd2fc2).
	* Abstract appeared in [RECOMB 2019](https://link.springer.com/content/pdf/bbm%3A978-3-030-17083-7%2F1.pdf).
0. Rachtman, Eleonora, Metin Balaban, Vineet Bafna, and Siavash Mirarab. “The Impact of Contaminants on the Accuracy of Genome Skimming and the Effectiveness of Exclusion Read Filters.” Molecular Ecology Resources, (2020): 1755-0998.13135. [doi:10.1111/1755-0998.13135](https://doi.org/10.1111/1755-0998.13135).
0. Rabiee, Maryam, and Siavash Mirarab. “Forcing External Constraints on Tree Inference Using ASTRAL.” BMC Genomics 21, no. S2 (2020): 218. [doi:10.1186/s12864-020-6607-z](https://doi.org/10.1186/s12864-020-6607-z).
0. Zhu, Qiyun, Uyen Mai, Wayne Pfeiffer, Stefan Janssen, Francesco Asnicar, Jon G. Sanders, Pedro Belda-Ferre, Al-Ghalith, Gabriel A., Evguenia Kopylova, Daniel McDonald, Tomasz Kosciolek, John B Yin, Shi Huang, Nimaichand Salam, Jian-yu Jiao, Zijun Wu, Zhenjiang Z. Xu, Kalen Cantrell, Yimeng Yang, Erfan Sayyari, Maryam Rabiee, James T Morton, Sheila Podell, Dan Knights, Wen-jun Li, Curtis Huttenhower, Nicola Segata, Larry Smarr, Siavash Mirarab, Rob Knight. “Phylogenomics of 10,575 Genomes Reveals Evolutionary Proximity between Domains Bacteria and Archaea.” Nature Communications 10, no. 1 (2019): 5477. [doi:10.1038/s41467-019-13443-4](https://doi.org/10.1038/s41467-019-13443-4).
0. One Thousand Plant Transcriptomes Initiative Green (a co-first author) “One Thousand Plant Transcriptomes and the Phylogenomics of Green Plants.” Nature, 2019. [doi:10.1038/s41586-019-1693-2](https://www.nature.com/articles/s41586-019-1693-2).
0. Balaban, Metin, Niema Moshiri, Uyen Mai, Xingfan Jia, and Siavash Mirarab. “TreeCluster: Clustering Biological Sequences Using Phylogenetic Trees.”  PLOS ONE 14, no. 8 (2019): e0221068. [doi:10.1371/journal.pone.0221068](https://doi.org/10.1371/journal.pone.0221068).
	* Presented at GLBIO 2019.
0. Houde, Peter, Edward L Braun, Nitish Narula, Uriel Minjares, and Siavash Mirarab. “Phylogenetic Signal of Indels and the Neoavian Radiation.” Diversity 11, no. 7 (2019): 108. [doi:10.3390/d11070108](https://doi.org/10.3390/d11070108). 
0. Rabiee, Maryam, and Siavash Mirarab. “INSTRAL: Discordance-Aware Phylogenetic Placement Using Quartet Scores.” Systematic Biology, early access (2019). [doi:10.1093/sysbio/syz045](https://doi.org/10.1093/sysbio/syz045).
0. Chen, Lei, Qiang Qiu, Yu Jiang, Kun Wang, Zeshan Lin, Zhipeng Li, Faysal Bibi, et al. “Large-Scale Ruminant Genome Sequencing Provides Insights into Their Evolution and Distinct Traits.” Science 364, no. 6446 (2019): eaav6202. [10.1126/science.aav6202](https://doi.org/10.1126/science.aav6202).
0. Yin, John, Chao Zhang, and Siavash Mirarab. “ASTRAL-MP : Scaling ASTRAL to Very Large Datasets Using Randomization and Parallelization.” Bioinformatics, btz211 (2019). [doi:10.1093/bioinformatics/btz211](https://doi.org/10.1093/bioinformatics/btz211)
0. Sayyari, Erfan, Ban Kawas, and Siavash Mirarab. “TADA: Phylogenetic Augmentation of Microbiome Samples Enhances Phenotype Classification.” Bioinformatics 35, no. 14 (2019): i31–40. [doi:10.1093/bioinformatics/btz394](https://academic.oup.com/bioinformatics/article/35/14/i31/5529256?guestAccessKey=4d5a3099-59c8-49ea-8da2-79d554774521)
	* As part of ISMB 2019.
0. Sarmashghi, Shahab, Kristine Bohmann, M. Thomas P. Gilbert, Vineet Bafna, and Siavash Mirarab. “Skmer: Assembly-Free and Alignment-Free Sample Identification Using Genome Skims.” Genome Biology 20, no. 1 (2019): 34. [doi:10.1186/s13059-019-1632-4](https://doi.org/10.1186/s13059-019-1632-4).
	* Abstract appeared in [RECOMB 2018](https://link.springer.com/content/pdf/bbm%3A978-3-319-89929-9%2F1.pdf).
0. Moshiri, Niema, Manon Ragonnet-cronin, Joel O Wertheim, and Siavash Mirarab. “FAVITES: Simultaneous Simulation of Transmission Networks, Phylogenetic Trees, and Sequences.” Bioinformatics (2018). [10.1093/bioinformatics/bty921](https://doi.org/10.1093/bioinformatics/bty921).
0. Rabiee, Maryam, Erfan Sayyari, and Siavash Mirarab. “Multi-Allele Species Reconstruction Using ASTRAL.” Molecular Phylogenetics and Evolution 130,  (2018): 286-96. [doi:10.1016/j.ympev.2018.10.033](https://doi.org/10.1016/j.ympev.2018.10.033).
0. Gopalakrishnan, Shyam, Mikkel-Holger S. Sinding, Jazmín Ramos-Madrigal, Jonas Niemann, Jose A. Samaniego Castruita, Filipe G. Vieira, Christian Carøe, et al. “Interspecific Gene Flow Shaped the Evolution of the Genus Canis.” Current Biology (2018). [doi:10.1016/j.cub.2018.08.041](https://doi.org/10.1016/j.cub.2018.08.041).
0. McDonald, Daniel, Embriette Hyde, Justine W. Debelius, James T. Morton, Antonio Gonzalez, Gail Ackermann, Alexander A. Aksenov, et al. “American Gut: An Open Platform for Citizen Science Microbiome Research.” mSystems, 3, no. 3 (2018). [doi.org/10.1128/mSystems.00031-18](https://doi.org/10.1128/mSystems.00031-18).
0. Janssen, Stefan, Daniel McDonald, Antonio Gonzalez, Jose A Navas-Molina, Lingjing Jiang, Zhenjiang Zech Xu, Kevin Winker, Deborah M Kado, Eric Orwoll, Mark Manary, Siavash Mirarab, Knight, Rob. “Phylogenetic Placement of Exact Amplicon Sequences Improves Associations with Clinical Information.” mSystems, 3 no. 3 (2018). [doi:10.1128/mSystems.00021-18](http://msystems.asm.org/content/3/3/e00021-18.abstract).
0. Zhang, Chao, Maryam Rabiee, Erfan Sayyari, and Siavash Mirarab. “ASTRAL-III: Polynomial Time Species Tree Reconstruction from Partially Resolved Gene Trees.” BMC Bioinformatics 19, no. S6 (2018): 153. [doi:10.1186/s12859-018-2129-y](https://doi.org/10.1186/s12859-018-2129-y).
0. Mai, Uyen, and Siavash Mirarab. “TreeShrink: Fast and Accurate Detection of Outlier Long Branches in Collections of Phylogenetic Trees.” BMC Genomics 19, no. S5 (2018): 272. [doi:10.1186/s12864-018-4620-2](https://doi.org/10.1186/s12864-018-4620-2).
0. Sayyari, Erfan, and Siavash Mirarab. “Testing for Polytomies in Phylogenetic Species Trees Using Quartet Frequencies.” Genes 9, no. 3 (2018): 132. [doi:10.3390/genes9030132](http://www.mdpi.com/2073-4425/9/3/132).
0. Eiserhardt, Wolf L., Alexandre Antonelli, Dominic J. Bennett, Laura R. Botigué, J. Gordon Burleigh, Steven Dodsworth, Brian J. Enquist, et al. “A Roadmap for Global Synthesis of the Plant Tree of Life.” American Journal of Botany 105, no. 3 (2018): 614–22. [doi:10.1002/ajb2.1041](https://doi.org/10.1002/ajb2.1041).
0. Sayyari, Erfan, James B. Whitfield, and Siavash Mirarab. “DiscoVista: Interpretable Visualizations of Gene Tree Discordance.” Molecular Phylogenetics and Evolution 122 (2018): 110–15. [doi:10.1016/j.ympev.2018.01.019](https://doi.org/10.1016/j.ympev.2018.01.019).
0. Akbari, Ali, Joseph J Vitti, Arya Iranmehr, Mehrdad Bakhtiari, Pardis C Sabeti, Siavash Mirarab, and Vineet Bafna. “Identifying the Favored Mutation in a Positive Selective Sweep.” Nature Methods 15, no 4 (2018): 279-282. [doi:10.1038/nmeth.4606](https://www.nature.com/articles/nmeth.4606).
0. Moshiri, Niema, and Siavash Mirarab. “A Two-State Model of Tree Evolution and Its Applications to Alu Retrotransposition.” Systematic Biology 67, no. 3 (2017): 475-489. [doi:10.1093/sysbio/syx088](https://doi.org/10.1093/sysbio/syx088).
0. Luke R. Thompson, Jon G. Sanders, Daniel McDonald, Amnon Amir, Joshua Ladau, Kenneth J. Locey, Robert J. Prill, Anupriya Tripathi, Sean M. Gibbons, Gail Ackermann, Jose A. Navas-Molina, Stefan Janssen, Evguenia Kopylova, Yoshiki Vázquez-Baeza, Antonio González, James T. Morton, Siavash Mirarab, Zhenjiang Zech Xu, Lingjing Jiang, Mohamed F. Haroon, Jad Kanbar, Qiyun Zhu, Se Jin Song, Tomasz Kosciolek, Nicholas A. Bokulich, Joshua Lefler, Colin J. Brislawn, Gregory Humphrey, Sarah M. Owens, Jarrad Hampton-Marcell, Donna Berg-Lyons, Valerie McKenzie, Noah Fierer, Jed A. Fuhrman, Aaron Clauset, Rick L. Stevens, Ashley Shade, Katherine S. Pollard, Kelly D. Goodwin, Janet K. Jansson, Jack A. Gilbert, Rob Knight & The Earth Microbiome Project Consortium. “A communal catalogue reveals Earth’s multiscale microbial diversity”. Nature 551 (2017): 457-463. [doi:10.1038/nature24621](http://www.nature.com/doifinder/10.1038/nature24621)
0. Sayyari, Erfan, James B Whitfield, and Siavash Mirarab. “Fragmentary gene sequences negatively impact gene tree and species tree reconstruction”. Molecular Biology and Evolution 34, no. 12 (2017): 3279–91. [doi:10.1093/molbev/msx261](http://dx.doi.org/10.1093/molbev/msx261).
0. Mai, Uyen, and Siavash Mirarab. “TreeShrink: Efficient Detection of Outlier Tree Leaves.” In RECOMB-CG 2017, Proceedings, 116–40. 2017. [doi:10.1007/978-3-319-67979-2_7](https://link.springer.com/chapter/10.1007/978-3-319-67979-2_7).
0. Zhang, Chao, Erfan Sayyari, and Siavash Mirarab. “ASTRAL-III: Increased Scalability and Impacts of Contracting Low Support Branches.” In RECOMB-CG 2017, Proceedings, 53–75. 2017. [doi:10.1007/978-3-319-67979-2_4](https://link.springer.com/chapter/10.1007/978-3-319-67979-2_4).
0. Mai, Uyen, Erfan Sayyari, and Siavash Mirarab. “Minimum Variance Rooting of Phylogenetic Trees and Implications for Species Tree Reconstruction.” PLOS ONE 12, no. 8 (2017): e0182238. [doi:10.1371/journal.pone.0182238](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0182238).
   * Best paper award at GLBIO 2017.
0. Mirarab, Siavash. “Phylogenomics: Constrained Gene Tree Inference”. Nature Ecology & Evolution (2017): 56. [doi:10.1038/s41559-016-0056](http://www.nature.com/articles/s41559-016-0056).
0. Shekhar, Shubhanshu, Sebastien Roch, and Siavash Mirarab. “Species Tree Estimation Using ASTRAL: How Many Genes Are Enough?”  IEEE/ACM Transactions on Computational Biology and Bioinformatics (2017). early access. [10.1109/TCBB.2017.2757930](http://ieeexplore.ieee.org/document/8053780/)  
   * Abstract [appeared](https://link.springer.com/content/pdf/bbm%3A978-3-319-56970-3%2F1.pdf) in  RECOMB 2017.
0. Sayyari, Erfan and Siavash Mirarab. “Anchoring quartet-based phylogenetic distances and applications to species tree reconstruction”. BMC Genomics 17, no. 10  (2016):  101–113. [doi:10.1186/s12864-016-3098-z](http://doi.org/10.1186/s12864-016-3098-z) 
0. Sayyari, Erfan and Siavash Mirarab. “Fast coalescent-based computation of local branch support from quartet frequencies”. Molecular Biology and Evolution 33, no. 7 (2016): 1654-1668. [doi:10.1093/molbev/msw079](https://doi.org/10.1093/molbev/msw079)
	* The supplementary material had a small error in one of its equations, fixed [here](assets/msw079-supplementary-v2.pdf)
{: reversed="reversed"}

### PhD work


{:start="34"}
0. Nguyen, Nam, Michael Nute, Siavash Mirarab and Tandy Warnow.  “HIPPI: Highly Accurate Protein Family Classification with Ensembles of HMMs”. BMC Genomics 17, no. 10 (2016): 89-100. [doi:10.1186/s12864-016-3097-0](http://doi.org/10.1186/s12864-016-3097-0).

0. Tarver, James E, Mario Dos Reis, Siavash Mirarab, Raymond J Moran, Sean Parker, Joseph E O’Reilly, Benjamin L King, Mary J O’Connell, Robert J Asher, Tandy Warnow, Kevin J Peterson, Philip C J Donoghue, and Davide Pisani. “The Interrelationships of Placental Mammals and the Limits of Phylogenetic Inference.” Genome Biology and Evolution (2016): evv261. [doi:10.1093/gbe/evv261](http://gbe.oxfordjournals.org/cgi/content/long/evv261v1).

0. Mirarab, Siavash, Md. Shamsuzzoha Bayzid, Bastien Boussau, and Tandy Warnow. 2015. “Response to Comment on ‘Statistical Binning Enables an Accurate Coalescent-Based Estimation of the Avian Tree.’” Science 350, no. 6257 (2015): 171. [doi:10.1126/science.aaa7719](http://www.sciencemag.org/content/350/6257/171.2.abstract).

0. Cracraft, Joel, Peter Houde, Simon Y W Ho, David P Mindell, Jon Fjeldså, Bent Lindow, Scott V Edwards, Carsten Rahbek, Siavash Mirarab,
Tandy Warnow, M Thomas P Gilbert, Guojie Zhang, Edward L Braun, Erich D Jarvis. “Response to Comment on ‘Whole-Genome Analyses Resolve Early Branches in the Tree of Life of Modern Birds.’” Science 349, no. 6255 (2015): 1460. [doi:10.1126/science.aab1578](http://www.sciencemag.org/content/349/6255/1460.2.abstract).

0. Jarvis, Erich D, Siavash Mirarab, Andre J Aberer, Bo Li, Peter Houde, Cai Li, Simon Y W Ho, et al. “Phylogenomic Analyses Data of the Avian Phylogenomics Project.” GigaScience 4, no. 1 (2015): 1–9. [doi:10.1186/s13742-014-0038-1](https://doi.org/10.1186/s13742-014-0038-1).

0. Davidson, Ruth, Pranjal Vachaspati, Siavash Mirarab, and Tandy Warnow. “Phylogenomic Species Tree Estimation in the Presence of Incomplete Lineage Sorting and Horizontal Gene Transfer.” BMC Genomics 16, Suppl 10 (2015): S1. [doi:10.1186/1471-2164-16-S10-S1](http://www.biomedcentral.com/1471-2164/16/S10/S1).

0. Chou, Jed, Ashu Gupta, Shashank Yaduvanshi, Ruth Davidson, Mike Nute, Siavash Mirarab, and Tandy Warnow. “A Comparative Study of SVDquartets and Other Coalescent-Based Species Tree Estimation Methods.” BMC Genomics 16, Suppl 10 (2015): S2. [doi:10.1186/1471-2164-16-S10-S2](http://www.biomedcentral.com/1471-2164/16/S10/S2).

0. Mirarab, Siavash, Tandy Warnow. “ASTRAL-II: Coalescent-Based Species Tree Estimation with Many Hundreds of Taxa and Thousands of Genes.”. Bioinformatics 31, no. 12 (2015): i44–i52. [doi:10.1093/bioinformatics/btv234](http://bioinformatics.oxfordjournals.org/content/31/12/i44)

0. Shamsuzzoha Md Bayzid, Siavash Mirarab, Bastien Boussau, and Tandy Warnow. “Weighted Statistical Binning: Enabling Statistically Consistent Genome-Scale Phylogenetic Analyses.” PLoS ONE 10, no. 6 (2015): e0129183. [doi:10.1371/journal.pone.0129183](http://dx.doi.org/10.1371/journal.pone.0129183).

0. Nam Nguyen, Siavash Mirarab, Keerthana Kumar, and Tandy Warnow. “Ultra-Large Alignments Using Phylogeny-Aware Profiles.”. Genome Biology 16, no. 1 (2015): 124. [doi:10.1186/s13059-015-0688-z](http://genomebiology.com/2015/16/1/124).
   * presented at RECOMB - 2015, Warsaw, Poland. Also appears as a one page abstract in the conference proceedings.
    
0. Mirarab, Siavash, Nam Nguyen, Sheng Guo, Li-San Wang, Junhyong Kim, and Tandy Warnow. “PASTA: Ultra-Large Multiple Sequence Alignment for Nucleotide and Amino-Acid Sequences.” Journal of Computational Biology  22, no. 05 (2015): 377–386. [doi:10.1089/cmb.2014.0156](http://online.liebertpub.com/doi/abs/10.1089/cmb.2014.0156)
	* an earlier version appeared at RECOMB 2014 proceedings (see below) 

0. Matasci, Naim, Ling-Hong L.-H. Hung, Zhixiang Yan, E.J. Eric J Carpenter, N.J. Norman J Wickett, Siavash Mirarab, Nam Nguyen, et al. “Data Access for the 1,000 Plants (1KP) Project.” GigaScience 3, no. 1 (2014): 17. [doi:10.1186/2047-217X-3-17](https://doi.org/10.1186/2047-217X-3-17).

0. Mirarab, Siavash, Md. Shamsuzzoha Bayzid, Bastien Boussau, and Tandy Warnow. “Statistical binning enables an accurate coalescent-based estimation of the avian tree.” Science 346, no. 6215 (2014). [doi:10.1126/science.1250463](http://www.sciencemag.org/content/346/6215/1250463.full)

0. Jarvis, Erich D, Siavash Mirarab (co-first author), Andre J Aberer, Bo Li, Peter Houde, Cai Li, Simon Y W Ho, et al. “Whole-Genome Analyses Resolve Early Branches in the Tree of Life of Modern Birds.” Science 346 , no. 6215 (2014): 1320–1331. [doi:10.1126/science.1253451](http://www.sciencemag.org/content/346/6215/1320.full)

0. Wickett, Norman J., Siavash Mirarab (co-first author), Nam Nguyen, Tandy Warnow, Eric Carpenter, Naim Matasci, Saravanaraj Ayyampalayam, Michael Barker, J. Gordon Burleigh, Matthew A. Gitzendanner, Brad R. Ruhfel, Eric Wafula, Joshua P. Der, Sean W. Graham, Sarah Mathews, Michael Melkonian, Douglas E. Soltish, Pamela S. Soltish, Nicholas W. Miles, Carl J. Rothfels, Lisa Pokorny, A. Jonathan Shaw, Lisa DeGironimo, Dennis W. Stevenson, Barbara Surek, Juan Carlos Villarreal, B ́eatrice Rourev, Herv ́e Philippe, Claude W. dePamphilis, Tao Chen, Michael K. Deyholos, Regina S. Baucom, Toni M. Kutchan, Megan M. Augustin, Jun Wang, Yong Zhang, Zhijian Tian, Zhixiang Yan, Xiaolei Wu, Xiao Sun, Gane Ka-Shu Wong, and James Leebens-Mack. A phylotranscriptomic analysis of the origin and early diversification of land plants. Proceedings of the National Academy of Sciences (PNAS), 111, no. 45 (2014): E4859–4868. [doi:10.1073/pnas.1323926111](http://www.pnas.org/content/early/2014/10/28/1323926111).

0. Mirarab, Siavash, Shamsuzzoha Bayzid, and Tandy Warnow. “Evaluating Summary Methods for Multi-Locus Species Tree Estimation in the Presence of Incomplete Lineage Sorting.” Systematic Biology (2014): [doi: 10.1093/sysbio/syu063](http://sysbio.oxfordjournals.org/content/early/2014/10/13/sysbio.syu063).

0. Mirarab, Siavash, Rezwana Reaz, Md. Shamsuzzoha Bayzid, Theo Zimmermann, M Shel Swenson, and Tandy Warnow. “ASTRAL: Genome-Scale Coalescent-Based Species Tree.” Bioinformatics 30, no. 17 (2014): i541–i548. [doi:10.1093/bioinformatics/btu462](https://doi.org/10.1093/bioinformatics/btu462).
	* presented at ECCB - 2014, Strasbourg, France 

0. Mirarab, Siavash, Nam Nguyen, and Tandy Warnow. “PASTA: Ultra-Large Multiple Sequence Alignment.” Proceedings of RECOMB (2014): 177–191. [doi:10.1007/978-3-319-05269-4_15](http://dx.doi.org/10.1007/978-3-319-05269-4_15).

0. Nguyen, Nam, Siavash Mirarab, Bo Liu, Mihai Pop, and Tandy Warnow. “TIPP: Taxonomic identification and phylogenetic profiling”. Bioinformatics (2014). [doi:10.1093/bioinformatics/btu721](http://bioinformatics.oxfordjournals.org/content/30/24/3548.full.pdf).

0. Zimmerman, Théo, Siavash Mirarab, and Tandy Warnow. “BBCA: improving the scalability of \*BEAST using random binning.” BMC Genomics 15, no. 6 (2014): S11. [doi:10.1186/1471-2164-15-S6-S11](http://www.biomedcentral.com/1471-2164/15/S6/S11) 

0. Stoltzfus, Arlin, Hilmar Lapp, Naim Matasci, Helena Deus, Brian Sidlauskas, Christian M Zmasek, Gaurav Vaidya, ... Siavash Mirarab, ... et al. “Phylotastic! Making Tree-of-Life Knowledge Accessible, Reusable and Convenient.” BMC Bioinformatics 14 (2013): 158. [doi:10.1186/1471-2105-14-158](http://www.biomedcentral.com/1471-2105/14/158).

0. Bayzid, M S, Siavash Mirarab, and Tandy Warnow. “Inferring Optimal Species Trees under Gene Duplication and Loss.” Pacific Symposium on Biocomputing (2013): 250–61. [PMID: 23424130](http://psb.stanford.edu/psb-online/proceedings/psb13/bayzid.pdf).

0. Mirarab, Siavash, Nam Nguyen, and Tandy Warnow. “SEPP: SATé-Enabled Phylogenetic Placement.” Pacific Symposium On Biocomputing (2012): 247–58. [PMID: 22174280](http://www.worldscientific.com/doi/pdf/10.1142/9789814366496_0024).

0. Nguyen, Nam, Siavash Mirarab, and Tandy Warnow. “MRL and SuperFine+ MRL: New Supertree Methods.” Algorithms for Molecular Biology 7, no. 1 (2012): 3. [doi:10.1186/1748-7188-7-3](http://www.almob.org/content/7/1/3/abstract).

0. Mirarab, Siavash, and Tandy Warnow. “FastSP: Linear Time Calculation of Alignment Accuracy.” Bioinformatics 27 (2011): 3250–8. [doi:10.1093/bioinformatics/btr553](http://bioinformatics.oxfordjournals.org/content/27/23/3250.long).
{: reversed="reversed"}

### MS work
{:start="9"}
0. Mirarab, Siavash, Soroush Akhlaghi, and Ladan Tahvildari. “Size-Constrained Regression Test Case Selection Using Multicriteria Optimization.” Software Engineering, IEEE Transactions on 38, no. 4 (2012): 936–956.

0. Do, Hyunsook, Siavash Mirarab, Ladan Tahvildari, and Gregg Rothermel. “The Effects of Time Constraints on Test Case Prioritization: A Series of Controlled Experiments.” Software Engineering, IEEE Transactions on 36, no. 5 (2010): 593–617.

0. Mirarab, Siavash, and Ladan Tahvildari. “An Empirical Study on Bayesian Network-Based Approach for Test Case Prioritization.” In Software Testing, Verification, and Validation, 2008 1st International Conference on, 278–287. IEEE, 2008.

0. Do, Hyunsook, Siavash Mirarab, Ladan Tahvildari, and Gregg Rothermel. “An Empirical Study of the Effect of Time Constraints on the Cost-Benefits of Regression Testing.” In Proceedings of the 16th ACM SIGSOFT International Symposium on Foundations of Software Engineering, 71–82. ACM, 2008.

0. Amoui, Mehdi, Mazeiar Salehie, Siavash Mirarab, and Ladan Tahvildari. “Adaptive Action Selection in Autonomic Software Using Reinforcement Learning.” In Autonomic and Autonomous Systems, 2008. ICAS 2008. Fourth International Conference on, 175–181. IEEE, 2008.

0. Mirarab, Siavash, Afshar Ganjali, Ladan Tahvildari, Shimin Li, Weining Liu, and Mike Morrissey. “A Requirement-Based Software Testing Framework: An Industrial Practice.” In Software Maintenance, 2008. ICSM 2008. IEEE International Conference on, 452–455. IEEE, 2008.

0. Mirarab, Siavash, Alaa Hassouna, and Ladan Tahvildari. “Using Bayesian Belief Networks to Predict Change Propagation in Software Systems.” In Program Comprehension, 2007. ICPC’07. 15th IEEE International Conference on, 177–188. IEEE, 2007.

0. Mirarab, Siavash, and Ladan Tahvildari. “A Prioritization Approach for Software Test Cases Based on Bayesian Networks.” In Fundamental Approaches to Software Engineering, 276–290. Springer Berlin Heidelberg, 2007.

