%______________________________________________________________________________________________________________________
% @brief    LaTeX2e Resume for Siavash Mirarab
\documentclass[margin,line,letterpaper]{resume}
\usepackage{setspace}
\usepackage{hyperref}
\usepackage{xr}

\externaldocument{publications}


\makeatletter
\renewenvironment{thebibliography}[1]
     {\list{\@biblabel{\@arabic\c@enumiv}}%
           {\settowidth\labelwidth{\@biblabel{#1}}%
            \leftmargin\labelwidth
            \advance\leftmargin\labelsep
            \@openbib@code
            \usecounter{enumiv}%
            \let\p@enumiv\@empty
            \renewcommand\theenumiv{\@arabic\c@enumiv}}%
      \sloppy
      \clubpenalty4000
      \@clubpenalty \clubpenalty
      \widowpenalty4000%
      \sfcode`\.\@m}
     {\def\@noitemerr
       {\@latex@warning{Empty `thebibliography' environment}}%
      \endlist}
\makeatother

\makeatletter
\renewcommand\@biblabel[1]{#1.}
\makeatother


\let\OLDthebibliography\thebibliography
\renewcommand\thebibliography[1]{
  \OLDthebibliography{#1}
  \setlength{\parskip}{1.5pt}
  \setlength{\itemsep}{1.6pt plus 0.3ex}
}
%_______________________________________________________________________________________
\begin{document}
\name{\Large Siavash Mirarab (Mir arabbaygi)}
\begin{resume}

    %_______________________________________________________________________________________    % Contact Information
%TODO: update address
    \section{\mysidestyle Contact\\Information}
UC San Diego (ECE)  ~ \hfill e-mail: \url{smirarab@ucsd.edu},	phone: 858 822 6245   \vspace{0mm}\\\vspace{0mm} 
%    \vspace{0mm} UC San Diego   \hfill \vspace{0mm}\\\vspace{0mm}
   9500 Gilman Drive, Mail code 0407  \hfill  \url{http://eceweb.ucsd.edu/~smirarab/}\vspace{0mm}\\\vspace{-4.5mm}%
   ~La Jolla, CA 92093-0407   	\hfill google scholar: \url{http://goo.gl/gEswQ5} \vspace{0mm}\\\vspace{-1.5mm}
   
      
%__________________________________________________________________________________________________________________
    % Research Interests
    \section{\mysidestyle Academic\\Positions}

    \textbf{UC San Diego (UCSD)}, San Diego, CA, United States\\
    Department of Electrical and Computer Engineering\vspace{1mm}\\\vspace{0mm}%
    \textsl{Assistant Professor} \hfill \textbf{ July 2015 -- July 2020}\vspace{0mm}\\%          
        \textsl{Associate Professor} \hfill \textbf{ July 2020 -- July 2025}\\
        \textsl{Full Professor} \hfill \textbf{ July 2025 -- present}\vspace{-3mm}\\
        \vspace{-4mm}%          
%__________________________________________________________________________________________________________________
    % Education
    \section{\mysidestyle Education\\ 
     ~~~~~~~~\textsl{Ph.D} \\  ~~\\ ~ \\~~~~~~~~~\textsl{M.S} \\  ~~\\ ~ \\~~~~~~~~~\textsl{B.S}}
    ~\\
    \textbf{University of Texas at Austin}, US  \hfill \textbf{ Jan. 2011 -- July 2015}\\\vspace{-4.5mm}%
    \begin{list2}
 %       \item GPA: 3.97
        \item Computer Science Department; Advisor:  Prof. Tandy Warnow (\& Keshav Pingali)
       % \item Novel scalable approaches for multiple sequencealignment and phylogenetic reconstruction
        %\item Research topics: large scale phylogenetics, metagenomics.
    \end{list2}\vspace{-1.5mm}
    
    \textbf{University of Waterloo},  ON, Canada \hfill \textbf{ Sept. 2006 -- Sept. 2008}\\\vspace{-4.5mm}%
%    \textbf{University of Waterloo}, Waterloo, Ontario, Canada,\\
%    Department of Electrical and Computer Engineering\vspace{-3mm}\\\vspace{-1mm}%
    \begin{list2}
 %       \item GPA: 91.6/100
        \item     Electrical and Computer Engineering;  Advisor:  Prof. Ladan Tahvildari
      %  \item Thesis: A Bayesian Framework for Software Regression Testing
    \end{list2}\vspace{-1.5mm}
%    \textbf{University of Tehran}, Tehran, Iran\\
 %   Department of Electrical and Computer Engineering \vspace{2mm}\\\vspace{1mm}%
    \textbf{University of Tehran}, Tehran, Iran \hfill \textbf{ Sept. 2001 -- Dec. 2005}\vspace{-3mm}\\\vspace{-1.5mm}%
    \begin{list2}
%        \item GPA: 79/100
        \item  Electrical and Computer Engineering (Specialized in software engineering)
        %\item Autonomous rescue agents: participation in 2003 Robocup simulation league.
    \end{list2}\vspace{-1.5mm}

%    %__________________________________________________________________________________________________________________
%    % Test Score
%    \section{\mysidestyle GRE Scores}
%	Verbal: 640 (92\%); Quantitative: 800 (94\%); Analytical Writing: 4 (41\%).

    %__________________________________________________________________________________________________________________
     %Honours and Awards
    \section{\mysidestyle Awards}
%TODO: add hackaton    
    %\begin{itemize}
    Clarivate Highly Cited Researcher \hfill \textbf{2022 -- 2024}\vspace{1mm}\\
    Best paper awards for SSB and RECOMB-CG, \hfill \textbf{2023}\vspace{1mm}\\
    Best student award runnerup for MBE \hfill \textbf{2021}\vspace{1mm}\\
    NIH-NIGMS MIRA award \hfill \textbf{ 2021}\vspace{1mm}\\
    NSF CAREER award \hfill \textbf{ 2019}\vspace{1mm}\\
    Alfred P. Sloan Research Fellow\hfill \textbf{ 2017}\vspace{1mm}\\
    Honorable Mention for the 2015 ACM Doctoral Dissertation Award.\hfill \textbf{ 2016}\vspace{1mm}\\
     Bert Kay Dissertation Award, UT,  Austin, Computer Science\hfill \textbf{ 2016}\vspace{1mm}\\
     Howard Hughes Medical Institute International Student Fellowship. \hfill \textbf{2012 - 2015}\vspace{1mm}\\
     National Science and Engineering Research Council of Canada (NSERC), 
      Canada Graduate Scholarship (CGS-D) converted to PGS-D outside Canada. \hfill \textbf{2011- 2012} \\
     Other awards: UT,  Austin, College of Natural Sci., Dean's Excellence Award (2011), Dean's Prestigious Fellowship Award (2011 --  2014), and MCD fellowship (2011) %(I declined to take CGS instead)
%     Travel awards from International Society for Computational Biology (ISCB) for ECCB, ISMB (2014, 2015),
%     National Science Foundation (NSF) for RECOMB (2014), 
%     National Library of Medicine/National Institutes of Health for PSB (2011, 2102),
%     Institute for Pure and Applied Mathematics (IPAM) for Mathematical and Computational Approaches in High-Throughput Genomic (2011),
%     and University of Texas at Austin, Dept. of Computer Science, for conference presentations (2011, 2012, 2013, 2014)
    
%\end{itemize}
    
    
    %__________________________________________________________________________________________________________________
    % Research Interests
    \section{\mysidestyle Research\\Funding}
     Minderoo Found.: ``Enabling the measurement of marine...''  \$726 K (2 years)~
     % Enabling the measurement of marine biodiversity with eDNA via metagenomics and targeted capture
 \hfill 2022\\
          NIH: R35GM142725:  ``Biology-aware machine learning...''
 \$1.7 M (5 years)~
  \hfill 2021\\
      NSF: CISE-1845967:  ``CAREER: Robust \& scalable genome...''
 \$549k (5 years)~
 \hfill 2019\\
      NSF: CISE-1815485 :  ``III: SMALL: New algorithms for genome...''
 \$500k (3 years)~
 \hfill 2018\\
 NSF: CISE-1565862:  ``CRII: III: Using genomic context to ...''
 \$170k (2 years)~
 \hfill 2016\\
 NIH-CFAR:  ``Accuracy of HIV transmission network ...''
 \$439K (2 years)~
 \hfill 2016\\
 CFAR, ``Algorithms for prioritizing prevention intervention ... HIV'', \$50K \hfill  ~2019--2020\\
  Sloan foundation fellowship, \$60K (2 years) \hfill  ~2016--2018\\
 Other gifts and internal grants $\approx$\$300K  \hfill  ~2017--2024



%\section{\mysidestyle Phd \\Students}
%%    \textbf{Graduated:}\vspace{2mm}\\\vspace{1mm}%
%Niema Moshiri, Now: Assistant Teaching Professor, UCSD (CSE Dep.)
%\hfill \textbf{2015 -- 2019}\\
%Erfan Sayyari:  Now: PostDoctral Researcher, UCSD (CMI)
%\hfill \textbf{2015 -- 2019}\\
%%\textbf{Graduated:}\vspace{2mm}\\\vspace{1mm}%
%Current: Metin Balaban, Yueyu Jiang, Uyen Mai, Maryam Rabiee, Nora Rachtman, Shahab Sarmashghi (co-superviser), Chao Zhang \hfill \textbf{2016~--~now}



    %__________________________________________________________________________________________________________________
    % Professional Experience
    \section{\mysidestyle Work Experience}

    \textsl{IBM}, Vancouver, BC, Canada
  %  Global Business Services (GBS) - Pacific Development Center (PDC)
%\vspace{2mm}\\\vspace{1mm}%
    \textsl{IT Specialist} \hfill \textbf{Sept. 2008 -- Dec. 2010}
 %   Worked on a software platform for implementing implantable wireless pace-maker devices for a large US-based biomedical company. 
%     Enterprise web application development using J2EE for health-care industry.                 \vspace{1mm}\\
%     Enterprise integration development using IBM WebSphere MQ and Massage Broker.               \vspace{1mm}\\
%     Database development in Oracle.

%    \textbf{Research In Motion (RIM)}, Waterloo, Ontario, Canada \vspace{2mm}\\\vspace{1mm}%
%    \textsl{Research Associate (part-time)} \hfill \textbf{Sept. 2007 -- Sept. 2008}  
%    Joint research project between RIM and University of Waterloo on methods of improving the testing process. \vspace{1mm}\\
%     Developed a probabilistic model for prioritizing regression test cases based on statistical information and requirement coverage data.

%    \textbf{Cisco Systems, ISBU}, San Jose, California, US \vspace{2mm}\\\vspace{1mm}%
%    \textsl{College Intern as Software Engineer} \hfill \textbf{May 2007 -- Sept. 2007}
%     Research on challenges of introducing service provider features in L2 switching devices.\vspace{1mm}\\
%     Learned Ciscoï¿½s platform, including IOS, and developed a new feature for Spanning Tree Protocol in order to improve its functionality for provider networks.


%    \textbf{Persian Java}, Tehran, Iran \vspace{2mm}\\\vspace{1mm}%
%    \textsl{Software Engineer (part-time)} \hfill \textbf{Jan. 2005 -- June 2005}
%     Requirement analysis and design using Rational Rose and UML.                \vspace{1mm}\\
%     Software architecture, focusing chiefly on Service Oriented Architecture
%     (SOA), integration with legacy systems, and work-flow engines.      
%     \vspace{1mm}\\ Built a web framework for rapid development of
%     workflow-backed JSP pages through a Struts-based control layer.

%    \textbf{Aviensystem}, Tehran, Iran \vspace{2mm}\\\vspace{1mm}%
%    \textsl{Software Engineer (part-time)} \hfill \textbf{July 2003 -- June 2005}       \\
%     Application development using Java, C\#, and PL/SQL, and J2EE web development.        \\
%     Built a framework for rapid application development in Java, following MVC paradigm, providing means of easily binding GUI (Swing) components to various data sources.                     \vspace{1mm}\\
%     R\&D: studying business rule engines, S.O.A, Aspect-Oriented Programming,
%     and BPEL; prototyping these technologies for the management.

%    \textbf{Kish Khodro}, Tehran, Iran \vspace{2mm}\\\vspace{1mm}%
%    \textsl{Software Development (project contract)} \hfill \textbf{Sept. 2001 -- June 2003}\\
%    Full development and maintenance of a total system for supply chain management.

%_______________________________________________________________________________________    %  Teaching Experience
    \section{\mysidestyle Teaching:}

    \textbf{Course Instructor (University of California, San Diego):}\vspace{2mm}\\\vspace{1mm}%
     Graduate: Computational evolutionary biology (ECE 208)  \hfill \textbf{2016 -- 2025}\\
     Introduction to Computer Engineering  (ECE 30) \hfill \textbf{2016 -- 2025}

\vspace{-10pt}
%%    
%%    %   \textbf{Other:}\vspace{2mm}\\\vspace{1mm}%
%%    Software Testing and Quality Assurance (TA), University of Waterloo.  \hfill \textbf{Winter 2007}\\
%%    Scientific \& Technical Communication (TA), University of Tehran.  \hfill \textbf{Fall 2005}\\
%%
%% \textbf{Workshops, Tutorials, and  Short Courses:}\vspace{2mm}\\\vspace{1mm}%
%    Ann Arbor, Michigan, Phylogenomics Symposium and Software School \hfill \textbf{2015}\\
%    UT Austin, Summer school on Phylogenomics and Metagenomics \hfill \textbf{Summer 2014}\\
%    Evolution meetings (NC), Software School on ASTRAL and PASTA \hfill \textbf{Summer 2014}\\
%    UT Austin, Workshop onPhylogeny and Alignment Estimation \hfill \textbf{Winter 2013}
%
%    \textbf{Guest Lecturer:}\vspace{2mm}\\\vspace{1mm}%
%University of Texas at Austin (Biology) Computation Biology \hfill \textbf{2014}\\
%University of Texas at Austin (CCBB) Introduction to Biological Computing \hfill \textbf{2014}\\
%University of Texas at Austin (Computer Science) Algorithm of Comp. Biology \textbf{2013}

%    \textbf{Volunteer Teaching:}\vspace{2mm}\\\vspace{1mm}%
%    Tehran, Iran, 12th grade mathematics and physics. \hfill \textbf{2001}
    
    
        \section{\mysidestyle Highlights and Invited Talks}
        
        See \url{http://eceweb.ucsd.edu/~smirarab/presentations.html} for a complete list\vspace{2mm}\\
        \href{https://www.imsi.institute/no_index/phylogenies-applications}{Institute for Mathematical and Statistical Innovation (IMSI)}, Chicago, IL \hfill \textbf{2025} \\
        \href{http://computationalgenomics.bioinformatics.ucla.edu/schedule/}{Computational Genomics Summer Institute (CGSI)}, Los Angles, CA\hfill \textbf{2017 -- 2025}\\
        Evolution Meeting, \href{https://www.evolutionmeetings.org/}{Athens, GA}, \href{https://www.evolutionsociety.org/tag/evolution-2023/}{Albuquerque, NM}\hfill \textbf{2023,~2025}\\
        \href{https://icerm.brown.edu/program/semester_program_workshop/sp-f24-w1}{Institute for Comp. and Exp. Research in Mathematics (ICERM)}, Rhode Island  \hfill \textbf{2024} \\
        \href{https://smbe2024.org/}{SMBE}, Puerto Vallarta, AI in genomics,  \hfill \textbf{2024} \\
       \href{https://ims.nus.edu.sg/events/mathematics_evolution/}{Mathematics of Evolution}, Singapore,  \hfill \textbf{2023} \\
       \href{https://recomb-seq.github.io/history/}{RECOMB-seq}, San Diego, keynote speaker, \hfill \textbf{2022} \\
       \href{https://cmsworkshops.com/Asilomar2022/Papers/ViewPapers.asp?PaperNum=1126&Mode=2}{Asilomar Conference on Signals, Systems, and Computers}, Monterey, \hfill \textbf{2022} \\
      % CGSI, UCLA, \hfill \textbf{2019 - 2023}\\
  % Bioinformatics Boot Camp for Ecology and Evolution, online\hfill \textbf{2020} \\
  \href{https://youtu.be/jYt28lTkAvg}{OH-Know!}, Norway (online),  \hfill \textbf{2021} \\
  \href{https://www.youtube.com/watch?v=pjCXxexKjus}{Phyloseminars}, online, \hfill \textbf{2021} \\
  Grainger Bioinformatics Center Symposium, online, \hfill \textbf{2020} \\
\href{https://twitter.com/sysbiodiv}{SysBioDiv} and Zoological Society, Freiburg, Germany \hfill \textbf{2020} \\
%\href{B10K}{}, Copenhagen, Denmark
%\href{ITA}{https://ita.ucsd.edu/ws/schedule2020/}, San Diego
      \href{https://uscbiscgrad.blogspot.com/2019/10/dr.html}{QCB Colloquium}, USC, Biological Sciences, Los Angles, CA\hfill \textbf{2019}\\
        \href{https://sites.google.com/view/mmm2019/home}{Workshop on Microbiomics, Metagenomics, and Metabolomics}, Buffalo, NY  \hfill \textbf{2019}\\
         \href{https://www.eventbrite.com/e/the-g10k-vgpebp-meeting-tickets-59112745814?aff=ebdshpsearchautocomplete}{Genome 10K \& Vertebrate Genome Project}, New York, NY \hfill \textbf{2019}\\
        Avian Meetings, Beijing, China,  San Diego, CA (with \href{https://pag.confex.com/pag/xxvii/meetingapp.cgi/Paper/36191}{PAG}) \hfill \textbf{2016, 2018, 2019}\\
        Evolution meeting,  \href{https://www.evolutionmeetings.org/uploads/4/8/8/0/48804503/evolution_2016_final_pdf_program.pdf}{Austin, TX},  \href{http://www.evolutionmeetings.org/uploads/4/8/8/0/48804503/final_evol2017_program.pdf}{Portland, OR}, \href{http://tandy.cs.illinois.edu/PhyloSynth-Symp2018.html}{Montpellier, France} \hfill \textbf{2016,~2017,~2018}\\
 \href{http://enable.nchu.edu.tw/international-page.php?uuid=f46f7f64-399e-11e9-97aa-309c231618d5}{ENABLE workshop},         National Chung Hsing University, Taiwan\hfill \textbf{2018}\\
        Institute for Pure \& Applied Math, Workshops on \href{http://www.ipam.ucla.edu/programs/workshops/multiple-sequence-alignment/?tab=speaker-list}{MSA} and \href{http://www.ipam.ucla.edu/programs/workshops/workshop-iii-hpc-for-computationally-and-data-intensive-problems/?tab=speaker-list}{HPC} \hfill \textbf{2015, 2018}\\
        ISMB, \href{https://www.iscb.org/ismb2018-program/ismb2018-tutorials#am4}{Invited Tutorial on comparative regulatory genomics}, Chicago, IL 
        %``Phylogenomic tree reconstruction'' 
        \hfill \textbf{2018}\\
        Simons Found., \href{https://indico.flatironinstitute.org/category/17/attachments/30/35/General_Agenda_SACMDA2_.pdf}{Workhop on Statistical Challenges in Microbiome}, Boston, MA %``Taxonomic Profiling using Scalable Phylogenetic Placement''
         \hfill \textbf{2017}\\
         University of Copenhagen, Seminar Series, Copenhagen, Denmark  \hfill \textbf{2017}\\
         Schloss Dagstuhl -- Leibniz Center for Informatics, \href{https://www.dagstuhl.de/no_cache/en/program/calendar/partlist/?semnr=16351&SUOG=}{16351}, Dagstuhl, Germany  \hfill \textbf{2016}\\
         RECOMB, \href{http://recomb2015.mimuw.edu.pl/node/6.html}{Highlights session}, Warsaw, Poland %``Statistical binning enables an accurate ...''   
         \hfill \textbf{2015}\\
      %   IPAM, UCLA, %``Ultra-large multiple sequence alignments''.\hfill \textbf{2015}\\
         INFORMS, \href{http://meetings2.informs.org/wordpress/philadelphia/files/2015/10/Monday.pdf}{Big Biological Data}, Philadelphia, PA %``Reconstruction of species trees using genomic data''.   
         \hfill \textbf{2015}\\
         MolPhy-3, Moscow, Russia %``SEPP and TIPP ...''. 
         \hfill \textbf{2012}\vspace{1mm}
%_______________________________________________________________________________________
    %  Professional Services
    \section{\mysidestyle Professional Services}
    %TODO: add summer schools. 
    Leadership: Vertebrate Genome Project (VGP), Council member.   \hfill \textbf{2020--present}\vspace{1mm}\\
    UCSD: Curriculum Development Committee. ``Engineering in Public Health''.\hfill \textbf{2020}\vspace{1mm} \\
     Conference organization: (IEEE BIBM) (2019), REOMB (2021)   \hfill \textbf{2019-2020}\vspace{1mm}\\
    Program Committee: ISBM, ECCB, RECOMB-CG, RECOMB  \hfill \textbf{2015, 2017--2024}\vspace{1mm}\\
    Paper Review for journals of wide readership (e.g., Science, Nature methods), in biology (e.g.,  Nature Ecology and Evolution), in math (e.g., Journal of the American Statistical Association), and in computer science (Journal of ACM). \\
    %See \url{https://publons.com/researcher/697671/siavash-mirarab} for a list.\\
 %   Includes papers in: Nature Ecology and Evolution, Nature Communications, Systematic Biology, PLOS Genetics, Journal of the American Statistical Association, Journal of ACM, IEEE Transaction on Computational Biology and Bioinformatics, Theoretical Population Genetics, Bioinformatics, BMC Bioinformatics, Genome Biology and Evolution, GigaScience    \hfill \textbf{2007-2019}\vspace{1mm}\\
Grant Panels:  NSF, DOE, NIH  \hfill \textbf{2018--2023}

    %Mentorship: Undergrad RA, University of Waterloo and UT Austin  \hfill \textbf{2008, 2013}
    %Mentorship: Undergrad Research Assistants at University of Waterloo   \hfill \textbf{2008}%







%__________________________________________________________________________________________________________________
    % Publications
    \renewcommand*{\thefootnote}{\fnsymbol{footnote}}
    \section{\mysidestyle Publications}

\newcommand\blfootnote[1]{%
  \begingroup
  \renewcommand\thefootnote{}\footnote{#1}%
  \addtocounter{footnote}{-1}%
  \endgroup
}
    
\def\FormatName#1{%
  \def\myname{Siavash Mirarab}
  \def\mynamee{Siavash Mirarab*}%
  \edef\name{#1}%
  \ifx\name\myname
   \underline{#1}%
  \else
    \ifx\name\mynamee
       \underline{Siavash Mirarab}*
    \else
      #1%and
    \fi
  \fi
}

\begin{footnotesize}
* indicates equal contribution
	\vspace{-20pt}
	%\renewcommand*{\refname}{\vspace*{-2em}}
	\begin{spacing}{-1}
	\input{output-subdoc/publications.bbl}
	%\blfootnote{* co-first authorship}
	\end{spacing}
\end{footnotesize}




\section{\mysidestyle Selected Software\\(Publicly\\available)}
\textbf{ASTRAL} (\url{https://github.com/smirarab/ASTRAL})\\
Coalescent-based species tree estimation from  gene trees\vspace{2mm}\\
\textbf{PASTA} (\url{https://github.com/smirarab/pasta})\\ 
Multiple sequence alignment of ultra-large datasets (up to a million sequences)\vspace{2mm}\\
\textbf{SEPP,TIPP,UPP} (\url{https://github.com/smirarab/sepp})\\
Phylogenetic placement and taxonomic identification of metagenomic reads\vspace{2mm}\\
%\textbf{TIPP} (\url{https://github.com/smirarab/sepp/blob/master/README.TIPP.md})\\
%Taxonomic profiling of metagenomic samples using phylogenetic placement\vspace{2mm}\\
%\textbf{UPP} (\url{https://github.com/smirarab/sepp/blob/master/README.UPP.md})\\
%Multiple sequences alignment using families of HMMs for ultra-large datasets\vspace{2mm}\\
\textbf{TreeShrink} (\url{https://github.com/uym2/TreeShrink})\\
Finding and removing spuriously long branches\vspace{2mm}\\
\textbf{DiscoVista} (\url{https://github.com/esayyari/DiscoVista})\\
Visualizing phylogenetic discordances\vspace{2mm}\\
\textbf{FAVITES} (\url{https://github.com/niemasd/FAVITES})\\
Simulating HIV evolution\vspace{2mm}\\
\textbf{APPLES} (\url{https://github.com/balabanmetin/apples})\\
Phylogenetic placement using distances \vspace{2mm}\\
\textbf{TreeCluster} (\url{https://github.com/niemasd/TreeCluster})\\
Clustering leaves of a phylogeny in linear time with gurantees
\vspace{2mm}\\
\textbf{DEPP} (\url{https://github.com/yueyujiang/DEPP})\\
Deep learning for phylogenetic placement
\vspace{2mm}\\
\textbf{ASTER} (\url{https://github.com/chaoszhang/ASTER})\\
Species tree estimation from  both gene trees and alignments \vspace{2mm}\\
\textbf{krepp} (\url{https://github.com/bo1929/krepp})\\
Read mapping to many genomes using $k$-mers \vspace{2mm}\\
%\textbf{INSTRAL} (\url{https://github.com/maryamrabiee/INSTRAL})\\
%Phylogenetic placement for full genomes\vspace{2mm}\\
%\textbf{FastSP} (\url{https://github.com/smirarab/FastSP})\\
%Fast comparison of very large multiple sequence alignments  
This list has recently grown too long to reproduce. 
See below for more. \\
\url{http://eceweb.ucsd.edu/~smirarab/software.html}

%______________________________________________________________________________________________________________________
\end{resume}
\end{document}


%______________________________________________________________________________________________________________________
% EOF

